perception/autoware_tensorrt_bevdet/include/autoware/tensorrt_bevdet/postprocess.hpp

```diff
@@ -0,0 +1,72 @@
+// Copyright 2024 AutoCore, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#ifndef AUTOWARE__TENSORRT_BEVDET__POSTPROCESS_HPP_
+#define AUTOWARE__TENSORRT_BEVDET__POSTPROCESS_HPP_
+
+#include "common.hpp"
+#include "iou3d_nms.hpp"
+
+#include <memory>
+#include <vector>
+
+class PostprocessGPU
+{
+public:
+  PostprocessGPU() {}
```

Comment:
The default constructor is empty. Consider initializing members to default values or explicitly stating that no initialization is needed if it's intentional.  This improves clarity.